### DISCUSSION TITLE
What is aparam.npy?

### DISCUSSION THREAD

**Discussion by psn417** [deepmodeling/deepmd-kit#184] [Q&A] [answered]
Dear all:
I use the data collected by dpgen to train a model, but I meet this error:
RuntimeError: ../data.init/al.POSCAR.01x01x01/02.md/sys-0004/deepmd/set.000/aparam.npy not found!
I already have box.npy coord.npy energy.npy force.npy virial.npy, but I don't have fparam.npy and aparam.npy. What are these two files and how to get them? I can't find any information in the manual.

S. Pan

**Comment by amcadmus** [comment]
please set `numb_aparam` to 0 or simply unset this key.

**Comment by psn417** [comment, marked as answer]
I check the code of dpgen and find I should use_ele_temp = 0 if I want to set aparam = 0. Thank you very much!
